src/svgdevicecontext.cpp

```diff
@@ -159,14 +159,6 @@ void SvgDeviceContext::StartGraphic(Object *object, std::string gClass, std::str
         m_currentNode.append_attribute("id") = gId.c_str();
     }
 
-    if (object->HasAttClass(ATT_VISIBILITY)) {
```

Comment:
I thought it would be nicer and a cleaner approach to have the attribute classes here in alphabetical order just like everywhere else, too. 
… and of course the svg attributes, as I tried to achieve.